INFO     log:test_init_phone.py:15 data===>[{'user_id': 1478}]
INFO     log:test_init_phone.py:17 1478
INFO     log:test_init_phone.py:21 随机数===>4m1pwg1szX
INFO     log:test_init_phone.py:22 随机更新的手机号===>136566402424m1pwg1szX
INFO     log:test_init_phone.py:26 ----------开始更新user表----------
INFO     log:mysql_operate.py:54 操作MySQL出现错误，错误原因：(1062, "Duplicate entry '136566402424m1pwg1szX' for key 'UK_OPEN_ID'")
INFO     log:test_init_phone.py:28 ----------执行更新user表成功----------
INFO     log:test_init_phone.py:29 ----------开始更新user_wechat_channel表----------
INFO     log:test_init_phone.py:31 ----------执行更新user_wechat_channel表成功----------
INFO     log:test_init_phone.py:32 ----------执行清除缓存的操作----------
INFO     log:redis_operate.py:26 ----------清除user_id为1478的token成功----------
INFO     log:redis_operate.py:27 2
INFO     log:redis_operate.py:30 ----------清除user_id为1478的tokens成功----------
INFO     log:test_init_phone.py:34 ----------执行清除缓存操作成功----------